src/platform/site-wide/cta-widget/index.js

```diff
@@ -444,6 +445,10 @@ export class CallToActionWidget extends React.Component {
       );
     }
 
+    const { appId, useSSOe } = this.props;
```

Comment:
If you put this block in the start of the `render` method you can delete lines 83-85, as the `render` method should get invoked before `componentDidUpdate`.  https://reactjs.org/docs/react-component.html#updating